frontend/src/components/chats/components/current-chat/components/message-form/message-form.tsx

```diff
@@ -25,16 +28,27 @@ const MessageForm: FC<Props> = ({ chatId, chatOpponentId }) => {
     payload: ChatMessageCreateRequestBodyDto,
   ): void => {
     dispatch(chatsActions.createMessage(payload));
+    reset();
   };
 
   return (
-    <form onSubmit={handleSubmit(handleMessageSubmit)}>
+    <form
+      onSubmit={handleSubmit(handleMessageSubmit)}
+      className={styles.messageForm}
+    >
       <Input
         placeholder="Your message here"
         control={control}
         errors={errors}
         name={getNameOf<ChatMessageCreateRequestBodyDto>('message')}
         label=""
```

Comment:
I propose to add some label + prop hasVisuallyHiddenLabel just not to render the empty span in dom